README.md

```diff
@@ -123,6 +123,8 @@ Swagger documentation generator for Fastify
 Templates rendering (*ejs, pug, handlebars, marko*) plugin support for Fastify.
 - [`fastify-orientdb`](https://github.com/mahmed8003/fastify-orientdb)
 Fastify OrientDB connection plugin, with this you can share the orientdb connection in every part of your server.
+- [`fastify-nats`](https://github.com/mahmed8003/fastify-nats)
```

Comment:
Please alphabetize the list. This item should be prior to `-orientdb` not after it.

Reply:
Also, it looks like another non-alphabetized commit got added. Would you mind moving `-orientdb` up to the correct position as well?